.clang-tidy

```diff
@@ -89,6 +91,16 @@ hicpp-*,
 # * rule: readability-use-anyofallof
 #   justification: requires C++20 and std::ranges but we only use C++14
 
+# * rule: bugprone-easily-swappable-parameters
+#   justification: Too restrictive as it does not allow consecutive data types
+#                  in interfaces that can be converted into each other.
+#   This is rather useless as the kinds of programming errors prevented are
+#   fairly special.
+#   We want to allow functions with signature
+#      void bar(int, int);
```

Comment:
This check is even harder and would also prevent interfaces like `void bar(int, float)` since `int` and `float` are implicitly convertible to each other.